### PR TITLE
Rockets actions

### PR DESCRIPTION
**Pull request by MostafaShahat2021**
## In This PR:
- ⭐Add actions and reducers for booking rockets.
- ⭐Add actions and reducers for canceling rockets.
- ⭐Add conditional components rendering.

### CODE REVIEW

File: src/Pages/rockets/Rockets.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            Reserve Rocket
+          </button>
+        )}
+        {rocket.reserved && (
+          <button
+            className="rocket-btn"
+            type="button"
+            onClick={() => dispatch(rocketCancel(rocket.id))}
+          >
+            Cansel Reservaition
```
Comment (NalianAchei2020) [inline src/Pages/rockets/Rockets.js:43]: ###  Optional:  
Please there in a typo in the spelling of cancel

**Review by NalianAchei2020 — APPROVED** [review]
# Status: Approved ✅✅
## Hi @MostafaShahat2021 ,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
- [ ] 

## Highlights
-  📌 No Linters error ✅
- 📌 Descriptive pull request✅
-  📌 Good commit messages ✅
- 📌 Clean code ✅




## Optional suggestions

- Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. 

- [ ] Cheers and Happy coding!👏👏👏

File: src/Pages/rockets/Rockets.js (hunk 1 above)
Comment (MostafaShahat2021) [reply src/Pages/rockets/Rockets.js:43]: Sorry 😅, I'll fix it, thanks ✌🏻

Comment (MostafaShahat2021) [reply src/Pages/rockets/Rockets.js]: Hey @NalianAchei2020 Thanks a lot 💯 for your efforts in reviewing my code, I really appreciate it 😄 .

**Pull request merged**